File: CS/Helpers/DragDrop/TreeListHelper.cs
using DevExpress.XtraGrid.Columns;
using DevExpress.XtraTreeList;
using System;
using System.Collections.Generic;
using System.Linq;

namespace dxQ529392 {
    public class TreeListHelper : BaseDropGridColumnHeaderSupportHelper<TreeList> {
        public override void DropObjectCore(string columnFieldName, System.Drawing.Point location, TreeList control) {
            control.Columns.AddField(columnFieldName).Visible = true;
        }
    }
}

File: VB/Helpers/DragDrop/TreeListHelper.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraGrid.Columns
Imports DevExpress.XtraTreeList
Imports System
Imports System.Collections.Generic
Imports System.Linq

Namespace dxQ529392
	Public Class TreeListHelper
		Inherits BaseDropGridColumnHeaderSupportHelper(Of TreeList)
		Public Overrides Sub DropObjectCore(ByVal columnFieldName As String, ByVal location As System.Drawing.Point, ByVal control As TreeList)
			control.Columns.AddField(columnFieldName).Visible = True
		End Sub
	End Class
End Namespace
